src/scripts/main.js

```diff
@@ -1 +1,70 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  let flag = false;
+
+  document.documentElement.addEventListener('click', () => {
+    flag = true;
+    resolve('First promise was resolved');
+  });
+
+  if (!flag) {
+    setTimeout(() => {
+      reject(new Error('First promise was rejected'));
+    }, 3000);
+  }
+});
+
+const secondPromise = new Promise(resolve => {
+  document.documentElement.addEventListener('click', () => {
+    resolve('Second promise was resolved');
+  });
+
+  document.documentElement.addEventListener('contextmenu', (e) => {
+    e.preventDefault();
+    resolve('Second promise was resolved');
+  });
+});
+
+const thirdPromise = new Promise(resolve => {
+  let leftClick = false;
+  let rightClick = false;
+
+  document.documentElement.addEventListener('mousedown', (e) => {
+    if (e.button === 0) {
+      leftClick = true;
+    }
+
+    if (e.button === 2) {
```

Comment:
I suggest to move magic numbers to constants like LEFT_MOUSE_BUTTON and RIGHT_MOUSE_BUTTON